server/server.js

```diff
@@ -17,21 +17,18 @@ app.use(express.json())
 
 app.use(express.urlencoded({ extended: true }))
 
-require('./routes/user.routes')(app)
-
-const PORT = process.env.PORT || 5000
-app.listen(PORT, () => {
-    console.log(`Server is runnig on port ${PORT}`)
-})
-
 const db = require('./models')
-
+// db.sequelize.drop()
 db.sequelize.sync().then(() => {
     console.log("Drop and re-sync db.");
   });
 
-app.get('/', async (req, res) => {
-    res.json({ message: "Welcome to postgresql application" })
+require('./routes/user.routes')(app)
+require('./routes/post.routes')(app)

```

Comment:
Why don't you group the routes with only one router, and use it in the server.js?
If you have more than 20 routers, server.js can be messy, your **model/index.js** is a good example

Reply:
you mean should I create routes/index.js to get all routes?